Build failed with an exception - Cannot locate tasks that match ':launchers:shadowJar' as task 'shadowJar'

# Bug Report

I followed the guide written in [README.md](https://github.com/eclipse-edc/FederatedCatalog/blob/main/README.md)
When I run command: `./gradlew :launchers:shadowJar`
The system returns following error:

```
ubuntu@gaia-x-2:~/git/FederatedCatalog$ ./gradlew :launchers:shadowJar

FAILURE: Build failed with an exception.

* What went wrong:
Cannot locate tasks that match ':launchers:shadowJar' as task 'shadowJar' not found in project ':launchers'.
```

### Expected Behavior

BUILD SUCCESSFUL should be returned

## Context Information
```
ubuntu@gaia-x-2:~/git/FederatedCatalog$ ./gradlew --version

------------------------------------------------------------
Gradle 8.5
------------------------------------------------------------

Build time:   2023-11-29 14:08:57 UTC
Revision:     28aca86a7180baa17117e0e5ba01d8ea9feca598

Kotlin:       1.9.20
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          17.0.11 (Ubuntu 17.0.11+9-Ubuntu-122.04.1)
OS:           Linux 5.15.0-112-generic amd64
```

```
ubuntu@gaia-x-2:~/git/FederatedCatalog$ java -version
openjdk version "17.0.11" 2024-04-16
OpenJDK Runtime Environment (build 17.0.11+9-Ubuntu-122.04.1)
OpenJDK 64-Bit Server VM (build 17.0.11+9-Ubuntu-122.04.1, mixed mode, sharing)
```

## Investigations
This issue doesn't occurs if I run: `./gradlew :launchers:catalog-dcp:shadowJar`

--- a/DEPENDENCIES
+++ b/DEPENDENCIES
@@ -93,7 +93,6 @@ maven/mavencentral/net.bytebuddy/byte-buddy-agent/1.14.1, Apache-2.0, approved, 
 maven/mavencentral/net.bytebuddy/byte-buddy-agent/1.14.15, Apache-2.0, approved, #7164
 maven/mavencentral/net.bytebuddy/byte-buddy/1.14.1, Apache-2.0 AND BSD-3-Clause, approved, #7163
 maven/mavencentral/net.bytebuddy/byte-buddy/1.14.15, Apache-2.0 AND BSD-3-Clause, approved, #7163
-maven/mavencentral/net.bytebuddy/byte-buddy/1.14.16, Apache-2.0 AND BSD-3-Clause, approved, #7163
 maven/mavencentral/net.bytebuddy/byte-buddy/1.14.18, Apache-2.0 AND BSD-3-Clause, approved, #7163
 maven/mavencentral/net.java.dev.jna/jna/5.13.0, Apache-2.0 AND LGPL-2.1-or-later, approved, #15196
 maven/mavencentral/net.sf.saxon/Saxon-HE/12.4, MPL-2.0 AND (MPL-2.0 AND Apache-2.0) AND (MPL-2.0 AND LicenseRef-X11-style) AND MPL-1.0 AND W3C, approved, #12716
@@ -313,15 +312,12 @@ maven/mavencentral/org.jetbrains/annotations/13.0, Apache-2.0, approved, clearly
 maven/mavencentral/org.jetbrains/annotations/17.0.0, Apache-2.0, approved, clearlydefined
 maven/mavencentral/org.jetbrains/annotations/24.1.0, Apache-2.0, approved, clearlydefined
 maven/mavencentral/org.junit-pioneer/junit-pioneer/2.2.0, EPL-2.0, approved, #11857
-maven/mavencentral/org.junit.jupiter/junit-jupiter-api/5.10.2, EPL-2.0, approved, #9714
 maven/mavencentral/org.junit.jupiter/junit-jupiter-api/5.10.3, EPL-2.0, approved, #9714
 maven/mavencentral/org.junit.jupiter/junit-jupiter-engine/5.10.3, EPL-2.0, approved, #9711
 maven/mavencentral/org.junit.jupiter/junit-jupiter-params/5.10.3, EPL-2.0, approved, #15250
-maven/mavencentral/org.junit.platform/junit-platform-commons/1.10.2, EPL-2.0, approved, #9715
 maven/mavencentral/org.junit.platform/junit-platform-commons/1.10.3, EPL-2.0, approved, #9715
 maven/mavencentral/org.junit.platform/junit-platform-engine/1.10.3, EPL-2.0, approved, #9709
 maven/mavencentral/org.junit.platform/junit-platform-launcher/1.10.3, EPL-2.0, approved, #15216
-maven/mavencentral/org.junit/junit-bom/5.10.2, EPL-2.0, approved, #9844
 maven/mavencentral/org.junit/junit-bom/5.10.3, EPL-2.0, approved, #9844
 maven/mavencentral/org.junit/junit-bom/5.9.2, EPL-2.0, approved, #4711
 maven/mavencentral/org.jvnet.mimepull/mimepull/1.9.15, CDDL-1.1 OR GPL-2.0-only WITH Classpath-exception-2.0, approved, CQ21484

--- a/README.md
+++ b/README.md
@@ -15,29 +15,32 @@ This repository contains all components needed to build and run a standalone ver
 
 ## Quick start
 
-A basic launcher configured with in-memory stores (i.e. no persistent storage) can be found [here](launchers/). There
-are
-two ways of running Federated Catalog:
+A basic launcher configured with in-memory stores (i.e. no persistent storage) can be
+found [here](launchers/catalog-mocked). Note that this runtime also contains a _mocked identity service_, for demo
+purposes. There is another variant called `catalog-dcp` which uses DCP as identity system and thus requires a properly
+configured DCP infrastructure to be set up. There are two ways of running Federated Catalog:
 
 1. As native Java process
 2. Inside a Docker image
 
-### Build the `*.jar` file
+### Build the `*.jar` files
 
 ```bash
-./gradlew :launchers:shadowJar
+./gradlew shadowJar
+# or
+./gradlew -p launchers shadowJar
 ```
 
 ### Start Federated Catalog as Java process
 
-Once the jar file is built, Federated Catalog can be launched using this shell command:
+Once the jar files are built, Federated Catalog can be launched using this shell command:
 
 ```bash
 java -Dweb.http.catalog.path="/api/catalog" \
      -Dweb.http.catalog.port=8181 \
      -Dweb.http.path="/api" \
      -Dweb.http.port=8080 \
-     -jar launchers/build/libs/fc.jar
+     -jar launchers/catalog-mocked/build/libs/fc-mocked.jar
 ```
 
 this will expose the Catalog API at `http://localhost:8181/api/catalog`. More information about Federated Catalog's APIs
@@ -46,7 +49,7 @@ can be found [here](docs/developer/architecture/federated-catalog-apis).
 ### Create the Docker image
 
 ```bash
-docker build -t fc ./launchers
+docker build -t fc ./launchers/catalog-mocked
 ```
 
 ### Start the Federated Catalog
@@ -60,7 +63,7 @@ docker run  --rm --name fc \
     fc:latest
 ```
 
-## Architectural concepts of IdentityHub
+## Architectural concepts of the Federated Catalog
 
 Key architectural concepts are
 outlined [here](docs/developer/architecture/federated-catalog.architecture.md).
@@ -74,7 +77,6 @@ detail [here](docs/developer/architecture/federated-catalog-apis.md).
 
 - Generalization of the Crawler class
 - Additional informational endpoints to the [Catalog API](docs/developer/architecture/federated-catalog.architecture.md)
-- PostgreSQL implementation for the `FederatedCacheStore`
 
 ## Other documentation
 

